pymc/distributions/discrete.py

```diff
@@ -1396,8 +1414,8 @@ class ZeroInflatedPoisson(Discrete):
 
     .. math::
 
-        f(x \mid \psi, \theta) = \left\{ \begin{array}{l}
-            (1-\psi) + \psi e^{-\theta}, \text{if } x = 0 \\
+        f(x \mid \psi, \mu) = \left\{ \begin{array}{l}
```

Comment:
For completeness should add this to release notes. And maybe even then we can just remove the depreciation warning.

Reply:
Yeah forgot the release notes